resources/sugarcrm/tests/sugarmockprotocol.cpp

```diff
@@ -234,3 +221,56 @@ void SugarMockProtocol::setContactsHandler(ContactsHandler *handler)
 {
     mContactHandler = handler;
 }
+
+void SugarMockProtocol::addData()
+{
+    mAccounts.resize(3);
```

Comment:
why not just call addAccounts() and addOpportunities() from here, instead of the duplication?